templates/minimal/zombienet-omni-node.toml

```diff
@@ -0,0 +1,15 @@
+[relaychain]
+default_command = "polkadot-omni-node"
+chain = "dev"
+chain_spec_path = "<path/to/chain_spec.json>"
+default_args = ["--dev-block-time 3000"]
+
+[[relaychain.nodes]]
+name = "alice"
+validator = true
+ws_port = 9944
+
+[[relaychain.nodes]]
+name = "bob"
+validator = true
+ws_port = 9955
```

Comment:
These two ZN files should be referenced in the main README as options, no? 

All in all, I see 5 options, perhaps it is good to put them in a ToC list somewhere: 

1. OmniNode
2. Old School Node
3. Old School Node + Docker 
4. ZN + Old School Node
5. ZN + OmniNode

Reply:
I would also move the Docker option to the bottom, as it seems least important to me.

Reply:
I structured once more the `minimal-template` README. I am hoping now it looks more complete and actionable: [ff56fc0](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/ff56fc0e3fd8f4f6680a4e38def8d6d9ca3d2efb). LMK your thoughts.